packages/amplify-provider-awscloudformation/src/__tests__/utils/amplify-resource-state-utils.test.ts

```diff
@@ -0,0 +1,61 @@
+import { getTableNames, getPreviousDeploymentRecord } from '../../utils/amplify-resource-state-utils';
+import { CloudFormation } from 'aws-sdk';
+
+const cfnClientStub = ({
+  describeStackResources: ({ StackName }) => ({
+    promise: () => ({
```

Comment:
I think this mock, and the one a few lines down should use `Promise.resolve()`.

Reply:
Thanks. Made the required changes 👍🏻